### ISSUE TITLE
Gradio button doesn't allow to put tags in value

### ISSUE THREAD

**Issue by HEYHEYHEEEEY** [gradio-app/gradio#8583]
### Describe the bug

Hello.
I am trying to add br tag in value of gradio button but without success. I have a simple code

```
gr.Button(value='Hi<br>New line', link='my_link')   
```

But gradio renders without new line
![image](https://github.com/gradio-app/gradio/assets/32372008/db0b1092-0f96-4364-9372-58e0789c506d)

A code of page
```
<a href="my_link" rel="noopener noreferrer" aria-disabled="false" class="lg secondary  svelte-cmf5ev" id="component-8" target="_blank"> Hi&lt;br&gt;New line</a>
```

Unfortunately, I can't find similar information about it. Why does gradio escape tags and replace them to &lt; and &gt; in value? How can I fix it?

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr

with gr.Blocks() as demo:
    gr.Button(value='Hi<br>New line', link='my_link')   

if __name__ == "__main__":
    demo.launch(show_api=False)
```

### System Info

```shell
gradio==4.32.2
```

### Severity

I can work around it

**Comment by pngwn** [comment]
Gradio component don’t generally accept arbitrary HTML and render it. You can use the HTML component for this or use some custom css to render line breaks.

**Comment by Isuxiz** [comment]
But I just want to allow line breaks in the button name... 
I don't think it's necessary to use HTML to simulate a button for this: it's too inefficient. 
I observed that the button component automatically escapes `<br />` to `&lt;br /&gt;`. Is it possible to refer to this logic and escape `\n` to `<br />`? I got the desired effect after manually modifying it in the Chrome dev console. I look forward to gradio officially providing this function. Thanks!
